### PR TITLE
groups vs forums and mentioning those

### PR DESCRIPTION
**Pull request by tobiasd**
for #1729 part of #1478

### CODE REVIEW

File: doc/Tags-and-Mentions.md [hunk 1]
```diff
@@ -21,11 +21,19 @@ You can tag a person on a different network or one that is **not in your social
 	
 * @mike@macgirvin.com - This is called a "remote mention" and can only be an email-style locator, not a web URL.
 
-Unless their system blocks unsolicited "mentions", the person tagged will likely receive a "Mention" post/activity or become a direct participant in the conversation in the case of public posts. Please note that Friendica blocks incoming "mentions" from people with no relationship to you. This is a spam prevention measure.
+Unless their system blocks unsolicited "mentions", the person tagged will likely receive a "Mention" post/activity or become a direct participant in the conversation in the case of public posts.
+Please note that Friendica blocks incoming "mentions" from people with no relationship to you.
```
Comment (annando) [inline doc/Tags-and-Mentions.md]: Have you tested if this really is the case?

Comment (tobiasd) [inline doc/Tags-and-Mentions.md]: In that section I've only applied the rule of "one sentence per line" but not changed the text.

But it was always the intended way to mention e.g. GNU Social accounts one does not follow directly so I assumed it is still valid.

Comment (annando) [inline doc/Tags-and-Mentions.md]: I think this sentence is not true for OStatus conversations.

Comment (tobiasd) [inline doc/Tags-and-Mentions.md]: How do you address OStatus people then if they pop-up in conversations but your are not following them?

Comment (tobiasd) [inline doc/Tags-and-Mentions.md]: I've just noted the @name@server.tld in a reply and hit the preview button, it got shortened to @name but that @name was also automatically added to the posting as usual when replying to OStatus people. I've not hit the send button though as a double mention makes no sense. But my guess is that the "good morning" round might offer a chance for testing this.

Comment (annando) [inline doc/Tags-and-Mentions.md]: I referred to "please note that Friendica blocks incoming "mentions" from people with no relationship to you."

I don't think that we are blocking that. Especially for OStatus I would deny that behaviour.

Comment (tobiasd) [inline doc/Tags-and-Mentions.md]: You should @mention me with one of your test accounts who is not connected with me then, just to see if this mention arrives me via OSatus then.

Comment (tobiasd) [inline doc/Tags-and-Mentions.md]: OStatus has been tested and the text updated accordingly.

**Pull request merged**